src/main/java/com/kodilla/ecommercee/domain/CartDto.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class CartDto {
+    private Long id;
+    private Product product;
```

Comment:
Klasy bazodanowe i DTO nie powinny mieć relacji między sobą. Jeśli cart ma zawierać Product to powinien to być ProductDto

Reply:
Przy okazji nie zgadza się to z twoim modelem na bazie. Tam masz Cart z wieloma produktami a tutaj zawsze z jednym